### PR TITLE
Temporary replace link to Luca's calendar with contact form

### PR DESCRIPTION
**Pull request by KevinBongart**
While Luca's on paternity leave, let's make sure people can still contact us for Rust help :)

This replaces the link to Luca's Calendly (https://calendly.com/luca-palmieri/is-rust-the-right-choice-for-us) with the generic Mainmatter contact form (https://mainmatter.com/contact/).

Other option: just `mailto:info@mainmatter.com`

### CODE REVIEW

File: src/rust-consulting.njk [hunk 1]
```diff
@@ -146,7 +146,7 @@ og:
         {%
           set link = {
           "label": 'Book a 1:1 call with Luca',
```
Comment (LukeMathWalker) [inline src/rust-consulting.njk]: You want to change the call to action here then.

Comment (LukeMathWalker) [reply src/rust-consulting.njk]: The whole paragraph needs to reworded, beyond the CTA itself. It's geared towards speaking with me, while now needs to be a "get in touch with our team" type of vibe.

Comment (KevinBongart) [reply src/rust-consulting.njk]: IMO people can reach out, we can say we have other Rust folks they can talk to while waiting for you to be back. We can talk about workshops you will run etc., it's still valuable to put your name forward.

How often do you get 1:1 calls via Calendly btw?

Comment (LukeMathWalker) [reply src/rust-consulting.njk]: Once a month or so? Sometimes more often, sometimes less. It depends on how much traffic the landing page gets, which in turn depends on what we released/advertised/etc.

This is how I'd reword the paragraph:

> Every new technology brings its own risks - Rust is no exception. Our team of in-house Rust experts is here to help you. You walk us through your company's usecase and your requirements and we'll do our best to suggest the best course of action. Sometimes Rust is not the answer, but if it is, we'll make sure you are on the right track.

CTA:

> Book a 1:1 call with us

Comment (LukeMathWalker) [reply src/rust-consulting.njk]: I'm already front-and-center in the leading section on that page:
image
Having me in the calendar call with language that strongly suggests you'd be speaking with _me_ is unnecessarily misleading imo.

**Comment by KevinBongart** [comment]
@LukeMathWalker Much better, thank you!

**Review by marcoow — APPROVED** [review]

**Comment by marcoow** [comment]
@KevinBongart #2512 fixes the unrelated linting issue

**Review by SaeLLo — APPROVED** [review]
